.gitignore

```diff
@@ -17,3 +17,7 @@ testbin/*
 
 # Kubernetes Generated files - skip generated files, except for vendored files
 !vendor/**/zz_generated.*
+
+# vscode files
```

Comment:
I prefer to keep these editor specific ignores out of the main repo, as others may be using different editors. If you're looking to do this across all your repos, I would suggest using a global gitignore git config option. See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer for details.